### PR TITLE
Dragging threshold

### PR DESCRIPTION
**Pull request by estshy**
Right now dragging starts on mousedown and touchstart which means it basically starts on click which is wrong. I am constantly moving elements on mobile when I am scrolling and toggling things. There should be dragging threshold (minimal number of pixels which element needs to be moved in order to be considered dragged).

### CODE REVIEW

File: addon/mixins/sortable-item.js [hunk 1]
```diff
@@ -119,6 +119,16 @@ export default Mixin.create({
    * @public
    */
   maxScrollSpeed: 20,
+  
+  /**
+   * How many pixes the element needs to be moved to be considered dragged
```
Comment (lukemelia) [inline addon/mixins/sortable-item.js]: `s/pixes/pixels/`

File: addon/mixins/sortable-item.js [hunk 2]
```diff
@@ -119,6 +119,16 @@ export default Mixin.create({
...
+   *
+   * @property draggingThreshold
+   * @type Number
+   * @default 15
+   * @public
+   */
+  draggingThreshold: 15,
```
Comment (lukemelia) [inline addon/mixins/sortable-item.js]: I think the default should probably be `1`

**Comment by lukemelia** [comment]
We've run into similar needs and were hoping something like this was available in ember-sortable.

File: addon/mixins/sortable-item.js (hunk 2 above)
Comment (estshy) [inline addon/mixins/sortable-item.js]: @lukemelia I guess it's hard to make a click without moving the mouse even by 1 pixels. I played with different values and that one came out as a good compromise.

In my option the value should block accidentally dragging by clicking for most of the situations  by default.

**Comment by acburdine** [comment]
@estshy This LGTM, and I will try to get this tested either today or tomorrow. Fixing @lukemelia's catch with the typo's probably not a bad idea though 😉

File: addon/mixins/sortable-item.js (hunk 1 above)
Comment (estshy) [inline addon/mixins/sortable-item.js]: @lukemelia Changed. Thanks.

**Comment by acburdine** [comment]
@estshy any chance of getting a rebase and some changes re: the above?

**Comment by mupkoo** [comment]
Some other libraries have `delay` option. Maybe this is something that can be added instead of the `draggingThreshold`?

**Comment by acburdine** [comment]
Forgot about this - closed by #137 as it fixes the issue present in this PR.

**Pull request closed without merging**